ci-operator/config/openshift/osd-network-verifier/openshift-osd-network-verifier-main.yaml

```diff
@@ -16,10 +15,6 @@ tests:
   commands: make test
   container:
     from: src
-- as: build
```

Comment:
if we remove build, how do we test `make build` to make sure verifier binary can be created anyway

Reply:
This commit was auto generated by `make prow-config`. I'm not sure what the logic behind removing `make build` is though.